tests/by-util/test_tail.rs

```diff
@@ -30,6 +37,462 @@ static FOLLOW_NAME_SHORT_EXP: &str = "follow_name_short.expected";
 #[allow(dead_code)]
 static FOLLOW_NAME_EXP: &str = "follow_name.expected";
 
+#[allow(dead_code)]
+struct TestRunner {
+    bytes: Option<String>,
+    lines: Option<String>,
+    follow: Option<String>,
+    retry: Option<String>,
+    sleep_interval: Option<String>,
+    max_unchanged_stats: Option<String>,
+    pid: Option<String>,
+    files: Vec<String>,
+    test_scenario: TestScenario,
+    child: Option<Child>,
+    stderr_to_stdout: bool,
+    pipe_in: Option<String>,
+    fifo_in: Option<std::fs::File>,
+    output: Option<tempfile::NamedTempFile>,
+    extra_args: Vec<String>,
+}
+
+impl Default for TestRunner {
+    fn default() -> Self {
+        Self {
+            bytes: None,
+            lines: None,
+            follow: None,
+            retry: None,
+            sleep_interval: None,
+            max_unchanged_stats: None,
+            pid: None,
+            files: vec![],
+            test_scenario: TestScenario::new(util_name!()),
+            child: None,
+            stderr_to_stdout: false,
+            pipe_in: None,
+            fifo_in: None,
+            output: None,
+            extra_args: vec![],
+        }
+    }
+}
+
+impl Drop for TestRunner {
+    fn drop(&mut self) {
+        if let Some(mut child) = self.child.take() {
+            let _ = child.kill();
```

Comment:
Is this not done automatically at `drop`? (Not implying that it is, I genuinely don't know)

Reply:
Yeah can't hurt I suppose